patricia_trie/src/node.rs

```diff
@@ -45,7 +91,7 @@ pub enum OwnedNode {
 	/// Extension node: partial key and child node.
 	Extension(NibbleVec, DBValue),
 	/// Branch node: 16 children and an optional value.
-	Branch([NodeKey; 16], Option<DBValue>),
```

Comment:
is `NodeKey` used anywhere else?

Reply:
> did you measure what was the impact of the performance if you only replaced ElasticArray with Vec? I'm just curious

No I didn't. There is neither `extend` nor `with_capacity` method for `ElasticArray`. I expect it to have bad performance, except if we build the vec on our side and use `from_vec` but it defeats the test.

In practice as I've said, I don't have a clear vision of what is the real situation, how big are all these collections? Do the bench make a good job at showing real scenarios?